### ISSUE TITLE
target/classes omitted when -testClasspath used

### ISSUE THREAD

**Issue by jglick** [johnpoth/jshell-maven-plugin#10]
By default, `target/classes` is included in the classpath. But if you specify `-testClasspath`, `target/test-classes` is added and `target/classes` is not present. This makes it impossible to script classes you are writing in your module!

**Comment by jglick** [comment]
You can use https://github.com/jglick/complex-maven-dependencies as an example.

```console
$ mvn verify
…
$ mvn -pl a com.github.johnpoth:jshell-maven-plugin:1.4-SNAPSHOT:run
[INFO] Scanning for projects...
[INFO] 
[INFO] -------------------------------< test:a >-------------------------------
[INFO] Building a 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- jshell-maven-plugin:1.4-SNAPSHOT:run (default-cli) @ a ---
|  Welcome to JShell -- Version 11.0.6
|  For an introduction type: /help intro

jshell> test.a.A.class.getProtectionDomain()
$1 ==> ProtectionDomain  (file:/…/jglick/complex-maven-dependencies/a/target/classes/ <no signer certificates>)
 jdk.internal.loader.ClassLoaders$AppClassLoader@55054057
 <no principals>
 java.security.Permissions@1a86f2f1 (
 ("java.io.FilePermission" "/…/jglick/complex-maven-dependencies/a/target/classes/-" "read")
 ("java.lang.RuntimePermission" "exitVM")
)



jshell> [INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  20.423 s
[INFO] Finished at: 2020-03-27T17:10:54-04:00
[INFO] ------------------------------------------------------------------------
$ mvn -pl a com.github.johnpoth:jshell-maven-plugin:1.4-SNAPSHOT:run -DtestClasspath
[INFO] Scanning for projects...
[INFO] 
[INFO] -------------------------------< test:a >-------------------------------
[INFO] Building a 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- jshell-maven-plugin:1.4-SNAPSHOT:run (default-cli) @ a ---
|  Welcome to JShell -- Version 11.0.6
|  For an introduction type: /help intro

jshell> test.a.A.class.getProtectionDomain()
|  Error:
|  cannot find symbol
|    symbol:   class A
|  test.a.A.class.getProtectionDomain()
|  ^------^

jshell> 
```

**Comment by johnpoth** [comment]
Fixed in 1.3
